source/locale/pt_BR/LC_MESSAGES/installation-guide.po

```diff
@@ -173,429 +217,236 @@ msgstr ""
 msgstr ""
 
-#: ../../source/installation-guide.rst:93
+#: ../../source/installation-guide.rst:105
 msgid "Docker Compose"
-msgstr "Docker-compose"
+msgstr "Docker Compose"
 
-#: ../../source/installation-guide.rst:95
+#: ../../source/installation-guide.rst:108
+msgid "This guide has been checked using Compose 1.27"
+msgstr "Este guia foi verificado utilizando o Compose na versão 1.27"
+
+#: ../../source/installation-guide.rst:110
 msgid ""
-"Up to date information and installation procedures for the docker-compose"
+"Up to date information and installation procedures for the Compose"
 " can be found at the project's documentation:"
 msgstr ""
-"Informações atualizadas sobre procedimentos de instalação para o docker-"
-"compose podem ser encontradas na documentação do projeto:"
+"Informações atualizadas sobre procedimentos de instalação para o "
+"Compose podem ser encontradas na documentação do projeto:"
 
-#: ../../source/installation-guide.rst:98
+#: ../../source/installation-guide.rst:113
 msgid "https://docs.docker.com/compose/install/"
 msgstr ""
 
-#: ../../source/installation-guide.rst:102
+#: ../../source/installation-guide.rst:116
 msgid "Installation"
 msgstr "Instalação"
 
-#: ../../source/installation-guide.rst:104
+#: ../../source/installation-guide.rst:119
 msgid ""
 "To setup the environment, merely clone the deployment repository and run "
 "the commands below."
 msgstr ""
 "Para construir o ambiente, simplesmente clone o repositório e execute os "
 "comandos abaixo."
 
-#: ../../source/installation-guide.rst:107
+#: ../../source/installation-guide.rst:122
 msgid ""
-"The docker-compose enabled deployment scripts and configuration "
+"The Docker Compose enabled deployment scripts and configuration "
 "repository can be found at:"
 msgstr ""
-"O repositório com os scripts de instalação e configuração do docker-"
-"compose podem ser encontrados em:"
+"O repositório com os scripts de instalação e configuração do Docker "
+"Compose pode ser encontrados em:"
 
-#: ../../source/installation-guide.rst:110
+#: ../../source/installation-guide.rst:125
 msgid "https://github.com/dojot/docker-compose"
 msgstr ""
 
-#: ../../source/installation-guide.rst:112
-msgid "or as git clone command:::"
-msgstr "ou com o comando git clone::"
+#: ../../source/installation-guide.rst:127
+msgid "or as git clone command: ::"
+msgstr "ou com o comando git clone:"
 
-#: ../../source/installation-guide.rst:119
+#: ../../source/installation-guide.rst:134
 msgid ""
 "Once the repository is properly cloned, select the version to be used by "
-"checking out the appropriate tag (do notice that the tagname has to be "
+"checking out the appropriate tag (do notice that the tag_name has to be "
 "replaced): ::"
 msgstr ""
 "Uma vez que o repositório esteja propriamente clonado, selecione a versão"
-" a ser utilizada por meio da tag apropriada (note que o tagname deve ser "
+" a ser utilizada por meio da tag apropriada (note que o tag_name deve ser "
```

Comment:
If you have changed the original sentence in English (`tag_name`), don't forget to format the translation too.